polyjuice/dicom_image.py

```diff
@@ -1,14 +1,24 @@
 import dicom
-
+import os
 class DicomImage(object):
 
     def __init__(self, dicom_file):
         self._dataset = dicom.read_file(dicom_file)
         self.filepath = dicom_file.name
 
+    def get_metadata(self, metadata_output):
+        _dataset = self._dataset
+        patient_id = self.get_patient_id()
```

Comment:
yeah dont make local variable that arent needed. The call is really short so just use that in place of a local copy